src/veil/ringct/lightwallet.h

```diff
@@ -0,0 +1,63 @@
+//
+// Created by main on 25/2/22.
```

Comment:
Clean up these comments. Replace with a standard copyright header seen in other files.